aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -1515,6 +1520,8 @@ def __init__(
             Each prefix can be a static string or a compiled regex pattern
         enable_validation: bool | None
             Enables validation of the request body against the route schema, by default False.
+        response_validation_error_http_status
+            Enables response validation and sets returned status code if response is not validated.
```

Comment:
```suggestion
        response_validation_error_http_status
            HTTP status code returned when response validation fails.
```

Response validation is already enabled, we are modifying the customer to inform a specific http status code.